wayback-core/src/main/java/org/archive/wayback/exception/RangeNotSatisfiable.java

```diff
@@ -0,0 +1,38 @@
+package org.archive.wayback.exception;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.archive.wayback.core.Resource;
+
+/**
+ * RangeNotSatisfiable is thrown when selected Resource does not have content data
+ * that can satisfy requested range.
+ * This happens when the Resource itself is a capture of range request that does not cover
+ * requested range.
+ * @see org.archive.wayback.replay.TransparentReplayRenderer
+ */
+public class RangeNotSatisfiable extends SpecificCaptureReplayException {
```

Comment:
Shouldn't this class be named 'RangeNotSatisfiableException'. I think that is the common convention for exceptions.